spec/lib/krikri/harvester_spec.rb

```diff
@@ -1,19 +1,61 @@
 require 'spec_helper'
 
 describe Krikri::Harvester do
-  it 'is a havester' do
-    expect(subject).to be_a Krikri::Harvester
-  end
+
+  # Our subject is an instance of a dummy class that mixes in
+  # Krikri::Harvester.
+  let(:klass) { Class.new }
+  subject { klass.include(Krikri::Harvester).new }
 
   context 'with record_ids implemented' do
     before do
-      described_class.send(:define_method, :record_ids) do
-        [1, 2, 3, 4, 5]
-      end
+      expect(subject).to receive(:record_ids).and_return([1, 2, 3, 4, 5])
```

Comment:
This is a significant improvement. :+1:

Maybe it's just an `allow` rather than an `expect`, though.  Do all possible tests in this context *need* to call `#record_ids`?